symphony-bdk-core/build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
     implementation 'io.swagger:swagger-annotations'
     implementation 'com.google.code.findbugs:jsr305'
     implementation 'org.glassfish.jersey.core:jersey-client'
+    implementation 'io.netty:netty-transport'
```

Comment:
To deal with import io.netty.channel.ConnectTimeoutException while using symphony-bdk-http-webclient

Reply:
Mmm we should not add it in the core, would it be possible to wrap it in the webclient? (in a SocketTimeoutException)

Reply:
I'm also wondering why we have the jersey client dependency

Reply:
the jersey client dependency is used for the same reason:
https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/symphony-bdk-core/src/main/java/com/symphony/bdk/core/retry/RetryWithRecoveryBuilder.java#L15
https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/AuthenticationRetry.java#L13

Reply:
For now I will wrap it in a RuntimeException with cause SocketTimeoutException to avoid making lot of changes inside the client code. This way we will have same result as for jersey2 client when timeout happens and we will be able remove the new dependency. 
Regarding the jersey one if it's ok I will prefer to handle it in the scope of this ticket: https://perzoinc.atlassian.net/browse/PLAT-10110